Trimble.Modus.Components/Controls/TMSlider/SliderCore.cs

```diff
@@ -52,11 +61,87 @@ static void OnLeftIconSourceChanged(BindableObject bindable, object oldValue, ob
             => ((SliderCore)bindable).OnLeftIconSourceChanged();
         static void OnRightIconSourceChanged(BindableObject bindable, object oldValue, object newValue)
             => ((SliderCore)bindable).OnRightIconSourceChanged();
+        static void OnToolTipBackgroundColorChanged(BindableObject bindable, object oldValue, object newValue)
+            => ((SliderCore)bindable).OnToolTipBackgroundColorChanged((Color)newValue);
+        static void OnTrackBackgroundColorChanged(BindableObject bindable, object oldValue, object newValue)
+            => ((SliderCore)bindable).OnTrackBackgroundColorChanged((Color)newValue);
+        static void OnTrackHighlightColorChanged(BindableObject bindable, object oldValue, object newValue)
+            => ((SliderCore)bindable).OnTrackHighlightColorChanged((Color)newValue);
+        static void OnThumbColorChanged(BindableObject bindable, object oldValue, object newValue)
+            => ((SliderCore)bindable).OnThumbColorPropertyChanged((Color)newValue);
+        private static void OnTitleTextColorChanged(BindableObject bindable, object oldValue, object newValue)
```

Comment:
If it just updates the color, can we do these with binding? as we are discussing in other PR's?